### ISSUE TITLE
enhancement: Clean up pacman cache

### ISSUE THREAD

**Issue by christianlupus** [AladW/aurutils#661]
I am unsure if this happened due to some weird misconfiguration, but I had the problem that there were *much* more packages in my pacman aur cache file (`/var/cache/pacman/aur` for me) then there should be. Looking closer it seems that pacman somehow put some cache files from other repositories there and also some older versions of aur packages were there.

I used my bash-foo and filtered out all packages that were not in the aur database (`aur.db.tar`) and removed those. However it would be nice to have a single extension to the aurutils command to cleanup the cache folder of pacman.

I could provide you the process/a script if needed but I fear that I cannot integrate it in the current version of the `aur` command. I doubt I get all the details correct...

**Comment by maximbaz** [comment]
This happened due to building in chroot via devtools, it won't happen once we get back to having aur-chroot in this repo (i.e. will not be an issue after #655, as far as I remember from my testing)

**Comment by christianlupus** [comment]
OK, then I will close this issue for now and reopen in case the problem rearises.

If anyone else has the same issue, please tell me, I will then reopen.
